atomate/feff/firetasks/__init__.py

```diff
@@ -1,5 +0,0 @@
-
-from .glue_tasks import *
-from .parse_outputs import *
-from .run_calc import *
-from .write_inputs import *
```

Comment:
Why were these removed?

Reply:
Oops. didn't look at the changes closely enough. This happened because of the option `--remove-all-unused-imports` flag passed to `autoflake` on [line 6 of `.pre-commit-config.yaml`](https://github.com/hackingmaterials/atomate/blob/main/.pre-commit-config.yaml#L6)

```yml
repos:
  - repo: git@github.com:humitos/mirrors-autoflake.git
    rev: v1.1
    hooks:
    - id: autoflake
      args: [ '--in-place', '--remove-all-unused-imports', '--remove-unused-variable' ]
```

To prevent similar mishaps in the future, I added the `--ignore-init-module-imports` flag. I also took the liberty of changing the repo link to the original `https://github.com/myint/autoflake` which previously linked to a mirror and running `pre-commit update` to update all 3 hooks.